pubtools/_ami/tasks/push.py

```diff
@@ -284,11 +288,12 @@ def update_rhsm_metadata(self, image, push_item):
             "arch": push_item.release.arch,
             "product_name": self.to_rhsm_product(
                 push_item.release.product, push_item.type
-            ),
+            )["name"],
             "version": push_item.release.version or None,
             "variant": push_item.release.variant or None,
         }
         LOG.info("Attempting to update the existing image %s in rhsm", image.id)
+        LOG.info(image_meta)
```

Comment:
Similarly as previous comment, if we're going to dump arbitrary python objects can we please have that at DEBUG level.